app/views/appointments/new.html.erb

```diff
@@ -55,7 +55,34 @@
         <div class="appointment">
           <input type="submit" value="Submit">
         </div>
-      </form>
+      </form> -->
+      <%= form_for @appointment do |f| %>
+        <div>
+        <%= f.label :first_name %>
```

Comment:
Then you need 2 pages: one for the administrator with all the appointments, and one for the user with only his